doc: Improves documention on how to obtain the `peering_id` for `mongodbatlas_network_peering` (#2488)
mongodb/terraform-provider-mongodbatlas: docs/resources/network_peering.md

$ terraform import mongodbatlas_network_peering.my_peering 1112222b3bf99403840e8934-5cbf563d87d9d67253be590a-AWS
```

> To obtain your `project_id` and `peering_id` with the (MongoDB Atlas CLI)[https://www.mongodb.com/docs/atlas/cli/current/command/atlas-networking-peering-list/#std-label-atlas-networking-peering-list]. Attention: The `atlas networking peering list` command returns `AWS` peerings by default. You must include the the `--provider` parameter: Name of your cloud service provider. Valid values are AWS, AZURE, or GCP. This value defaults to "AWS".

Reviewer: This removes a typo and improves readability a bit.

```suggestion 
Use the [MongoDB Atlas CLI][https://www.mongodb.com/docs/atlas/cli/current/command/atlas-networking-peering-list/#std-label-atlas-networking-peering-list] to obtain your `project_id` and `peering_id`. Attention gcp and azure users: The `atlas networking peering list` command returns only `AWS` peerings by default. You have to include the `--provider` parameter to list peerings for your cloud provider. Valid values are AWS, AZURE, or GCP. This value defaults to "AWS".
```

> Use the [MongoDB Atlas CLI][https://www.mongodb.com/docs/atlas/cli/current/command/atlas-networking-peering-list/#std-label-atlas-networking-peering-list] to obtain your `project_id` and `peering_id`. Attention gcp and azure users: The `atlas networking peering list` command returns only `AWS` peerings by default. You have to include the `--provider` parameter to list peerings for your cloud provider. Valid values are AWS, AZURE, or GCP. This value defaults to "AWS".

Reviewer: Suggestion to remove the redundant mentioning of default option.

```suggestion
Use the [MongoDB Atlas CLI][https://www.mongodb.com/docs/atlas/cli/current/command/atlas-networking-peering-list/#std-label-atlas-networking-peering-list] to obtain your `project_id` and `peering_id`. Attention gcp and azure users: The `atlas networking peering list` command returns only `AWS` peerings by default. You have to include the `--provider` parameter to list peerings for your cloud provider. Valid values are AWS, AZURE, or GCP.
```

Reviewer 2: +1
